src/lib/gameService.ts

```diff
@@ -12,41 +13,172 @@ export class GameService {
 			code: this.generateCode(),
 			currentPlayer: undefined,
 			players: [],
-			currentBid: undefined
+			currentBid: undefined,
+			maxPlayers: 8,
+			numberOfPlayers: 0,
```

Comment:
`numberOfPlayers` probably isn't needed, as it's easily derived from the players array (and we might forget to keep it in sync with the actual player count)